main.go

```diff
@@ -142,7 +142,7 @@ func main() {
 		},
 		&cli.StringFlag{
 			Name:    "safe-directory",
-			Usage:   "Define safe directories",
+			Usage:   "Define / replace safe directories",
```

Comment:
```suggestion
			Usage:   "Define/replace safe directories",
```

Small one, use same format as in Line 135